uploadNeuroDB/tarchive_validation.pl

```diff
@@ -15,6 +15,8 @@ =head1 SYNOPSIS
 
 -profile     : name of the config file in C<../dicom-archive/.loris-mri>
 
+-uploadID    : UploadID associated to the DICOM archive to validate
```

Comment:
Is there a reason to have 3 different texts for the uploadID  param in theses 3 files?

Apart that, it look good to me. Note that I'am no expert with LORIS-MRI

Reply:
@PapillonMcGill It is slightly different because each script using that option is at a different step in the pipeline (generic upload, DICOM archive validation or MINC insertion all derived from the same upload). Hope this makes more sense now.

Reply:
@cmadjar Yes it does. 
Thanks!